recipes/implot/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
```

`test_v1_package` must still use old v1 generators

```suggestion

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

```

---

:+1: thats the point of it

---

Done in [4eae8ad](https://github.com/conan-io/conan-center-index/pull/16847/commits/4eae8ad839009191836b0c2d4d2cf5482a87fa16)